server/game/cards/characters/06/knightofthereach.js

```diff
@@ -0,0 +1,37 @@
+const DrawCard = require('../../../drawcard.js');
+
+class KnightOfTheReach extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: (e, challenge) => (
+                    challenge.winner === this.controller && 
+                    challenge.isParticipating(this) &&
+                    this.hasSingleParticipatingChar())
+            },
+            target: {
+                activePromptTitle: 'Select a Lady',
+                cardCondition: card => (
+                    card.location === 'play area' &&
+                    card.hasTrait('Lady') &&
+                    card.getType() === 'character')
+            },
+            handler: context => {
+                context.target.modifyPower(1);
+                this.game.addMessage('{0} uses {1} to have {2} gain 1 power', 
+                                      this.controller, this, context.target);
+            }
+        });
+    }
+
+    hasSingleParticipatingChar() {
```

Comment:
This is a candidate for moving on to the challenge itself if this starts becoming a common thing